packages/gatsby-transformer-remark/src/__tests__/__snapshots__/extend-node.js.snap

```diff
@@ -583,7 +583,9 @@ Object {
 <li><a href=\\"/my%20little%20pony/#second-title\\">second title</a></li>
 </ul>
 </li>
-<li><a href=\\"/my%20little%20pony/#third-title\\">third title</a></li>
+<li>
```

Comment:
Bumping `mdast-util-to-hast` to its latest version caused this small change in the generated HTML. If this is an unacceptable regression, we can avoid bumping that dependency at this time and just bump `mdast-util-toc` which doesn't regress any existing tests.

Reply:
This is tough question - the change in this PR seems like more sane markup than what currently is generated, but it can cause issues to users potentially.

If possible, let's try to keep current markup for now, so we can merge bugfix, without long discussions about this?

Reply:
That's understandable.  I've reverted the version bump and reverted these test changes as part of the most recent commit.